bench/commands/__init__.py

```diff
@@ -70,4 +70,10 @@ def bench_command(bench_path='.'):
 bench_command.add_command(remote_urls)
 
 from bench.commands.install import install
-bench_command.add_command(install)
\ No newline at end of file
+bench_command.add_command(install)
+
+from bench.commands.exclude_update import exclude_update
+bench_command.add_command(exclude_update)
+
+from bench.commands.exclude_update import include_update
+bench_command.add_command(include_update)
```

Comment:
Also name rename the methods to `include_app_for_update` and `exclude_app_for_update`